main-app/app/scripts/bingoTicket/bingoModel.js

```diff
@@ -20,14 +20,29 @@
                 }
             };
 
+            var BingoCell = function () {
+                this.bingoNumber = null;
+                this.marked = "unmarked";
```

Comment:
this.marked = 'unmarked' is used for a class. If the number is found this.marked = 'checked' which colours in the bingo cell